jawn-core/src/main/java/net/javapla/jawn/core/templates/ContentTemplateLoader.java

```diff
@@ -29,10 +34,13 @@ public final String getTemplateRootFolder() {
     public final String handleLayoutEndings(Result response) {
         String layout = response.layout();
         if (layout != null) {
-            if (layout.endsWith(templateSuffix)) //TODO is it faster to keep it as a reference?
-                layout = layout.substring(0, layout.length()-3);
```

Comment:
Arh.. I see what you did there..
The '-3' is a relic from a refactoring, that apparently was not done correctly